apps/balancer-tools/src/app/apr/pool/(components)/HistoricalAPRPlot.tsx

```diff
@@ -0,0 +1,26 @@
+"use client";
```

Comment:
👌  great use of the "use client" directive

Reply:
Thank you! I was somewhat unsure about it because I hadn't seen a component split like this yet.